src/payment/PaymentProcessingModal.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
+
+import {
+  ModalDialog, Spinner,
+} from '@edx/paragon';
+import { useSelector } from 'react-redux';
+import { useIntl, defineMessages } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+
+// import { subscriptionStatusSelector } from '../data/status/selectors';
+
+const messages = defineMessages({
```

Comment:
nit: separate into PaymentProcessingModal.messages.jsx

Reply:
Ill do this morning